### DISCUSSION TITLE
React Hook Form: Form state(dirtyFields) - append function triggering second form input update

### DISCUSSION THREAD

**Discussion by Kiptim54** [react-hook-form/react-hook-form#2676] [Q&A]
Hey, I am new to react hook forms and I am having a bit of an issue. 

I am using the formState to track changes made to the various inputs to update my backend. However,  when I trigger the append function to add a new form, the already existing second input is added to the dirty fields and it seems like a user has updated it even when this is not the case. 

Here is a sandbox that illustrates this: https://codesandbox.io/s/recursing-cloud-dtpql?file=/src/App.js

Any assistance with this will be greatly appreciated, thanks

**Comment by bluebill1049** [comment]
same: https://spectrum.chat/react-hook-form/help/react-hook-form-form-state-dirtyfields-append-function-triggering-second-form-input~aae09691-e82d-4ab6-9442-3a885d80e319
